core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -444,6 +444,9 @@ ExHbaseAccessTcb::ExHbaseAccessTcb(
   rows_.len = 0;
   colVal_.val = 0;
   colVal_.len = 0;
+  asyncCompleteRetryCount_ = 0;
+  asyncOperationTimeout_ = 2;
```

Comment:
Timeout increases helps to avoid cpu hogging when RS takes more time to respond. Yes. we will plan for a CQD later to provide a maximum timeout value. I would suggest avoiding coupling it with scanner timeout